src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/KravgrunnlagService.kt

```diff
@@ -131,18 +133,36 @@ class KravgrunnlagService(
         tellerService.tellKobletKravgrunnlag(fagsystem)
     }
 
-    private fun skalBehandlesAutomatisk(
+    fun kanBehandlesAutomatiskBasertPåRettsgebyrOgfagsystemreferanse(
         kravgrunnlag431: Kravgrunnlag431,
         behandling: Behandling,
-    ) = erUnder4xRettsgebyr(kravgrunnlag431) && behandlingOgKravgrunnlagReferererTilSammeFagsystembehandling(behandling, kravgrunnlag431)
+    ): Boolean =
+        kanBehandlesAutomatiskBasertPåRettsgebyr(kravgrunnlag431) &&
+            behandlingOgKravgrunnlagReferererTilSammeFagsystembehandling(behandling, kravgrunnlag431)
+
+    fun erOverFireRettsgebyr(behandling: Behandling): Boolean {
+        val kravgrunnlag = kravgrunnlagRepository.findByBehandlingIdAndAktivIsTrue(behandling.id)
+        return !kanBehandlesAutomatiskBasertPåRettsgebyr(kravgrunnlag)
+    }
+
+    private fun kanBehandlesAutomatiskBasertPåRettsgebyr(
```

Comment:
```suggestion
    private fun feilutbetalingErUnderFireGangerRettsgebyr(
```

Dette er hva metoden faktisk sjekker. Som utenforstående forstår ikke jeg uten videre at dette også betyr at man automatisk kan behandle tilbakekrevingen basert på rettsgebyr.

Reply:
Vi endrer til feilutbetalingErUnderFireRettsgebyr og legger på en kommentar for å forklare quirk (catch -> false)